src/components/App.js

```diff
@@ -3,33 +3,34 @@ import {
   Route,
   Routes,
 } from 'react-router-dom';
-
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-
-import { getMissions } from '../redux/missions/missionsSlice';
```

Comment:
I also noticed that you removed getMissions import.
I understand that it caused conflicts but it seems the conflict wasn't solved properly.
Could you please reimport { getMissions} as it's an important part of my part of work 🙏🏻